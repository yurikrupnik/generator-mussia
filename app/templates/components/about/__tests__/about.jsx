import React from 'react';
import About from '../index';

const {
    it,
    expect,
    shallow
} = global;

it('renders three <About /> components', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
});
