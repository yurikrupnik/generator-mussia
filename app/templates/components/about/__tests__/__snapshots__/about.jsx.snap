// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders three <About /> components 1`] = `
<div
  className="about"
>
  <h2>
    About
  </h2>
</div>
`;
